tls/s2n_config.c

```diff
@@ -167,23 +170,37 @@ static int s2n_config_update_domain_name_to_cert_map(struct s2n_map *domain_name
         struct domain_name_to_cert_map_value *value = (void *) s2n_map_value.data;;
         if (value->certs[auth_method] == NULL) {
             value->certs[auth_method] = cert_key_pair;
+        } else if (config->cert_tiebreak_cb) {
```

Comment:
If the application does not implement a tiebreak_cb, then there is no way to update the existing cert in the map for that domain. The only other way is to create a new s2n_config and re-add the certificates that the application wants. Is this correct? May want to document this behavior in the USAGE_GUIDE.

Reply:
added comment that first certificate added will always be preferred